Firmware/Marlin_main.cpp

```diff
@@ -8834,10 +8852,15 @@ SERIAL_PROTOCOLPGM("\n\n");
     }
     break;
 
-  /*!
-  ### M701 - Load filament <a href="https://reprap.org/wiki/G-code#M701:_Load_filament">M701: Load filament</a>
-  
-  */
+    /*!
+    ### M701 - Load filament <a href="https://reprap.org/wiki/G-code#M701:_Load_filament">M701: Load filament</a>
+    #### Usage
+
+        M701 [ E ]
```

Comment:
In RepRap gcode wiki it is https://reprap.org/wiki/G-code#M701:_Load_filament

Do we need to update the wiki for PrusaFimware using `E` instead of `T`?

Reply:
Hmmm. Now that's a stupid inconsistency from marlin.... Makes me wonder if we should duplicate the code_seen() so that T is also supported

Reply:
I propose we merge this PR (documentation only) and then I add the T parameter alias as well in another PR. It shouldn't break any compatibility

Reply:
Change of plans. I just added the commit here since the change is really small. Tested the change on the old mmu implementation